pushconfig.sh

```diff
@@ -380,13 +385,22 @@ function update_configuration() {
 
     selector=$1
     api_port=$2
-    config_files=$(find -L $FOLDER -iname "*.json" -type f -printf "%p\n" | sort)
+    config_files=$(find -L $FOLDER \( -iname "*.json" ! -iname "*_${CONFIG_VARIANT}.json" \) -type f -printf "%p\n" | sort)
```

Comment:
Where is defined ${CONFIG_VARIANT} as this point ?

Reply:
Bien vu, CONFIG_VARIANT est issu de l'env de PRL pour spécifier la variante de configuration à utiliser. Elle sert à positionner le `-o` du script `pushconfig` et donc `EXTENSION_OVERRIDE`.
Je corrige ça